packages/client/Makefile

```diff
@@ -0,0 +1,9 @@
+# DU client requires the types generated from DU contracts
+unit-test: ../contracts/typechain
+	npx ganache -p 3456 -m testrpc &
+	npx jest --runInBand test/unit
+	exit 0 # terminate ganache
```

Comment:
I haven't tested, but I don't think running `exit 0` is going to kill ganache? Can you double check what happens here?

Reply:
seems it gets terminated (since test run ends), but I'm not 100% how makefiles work here. In a regular script, `exit 0` would terminate the script (shell?) and the ganache process with it.

Reply:
hmm actually true; it doesn't get terminated. I'll fix that.

Reply:
You could do something like (but this might get tricky):
```
ganache &
GANACHE_PID=$$!
# run test
kill -SIGTERM $(GANACHE_PID)
```
Or run ganache as a GA service container https://docs.github.com/en/actions/using-containerized-services/about-service-containers
(I'm not sure if it is possible at all)

Reply:
yea it gets tricky due to how Makefiles work. Seems `a && b` and
```
a
b
```
are two very different things in Makefiles  :/  but I'll get used to it.